Logger.go

```diff
@@ -110,10 +112,13 @@ func (l *logger) IsLevelEnabled(level Level) bool {
 	return l.Logger.IsLevelEnabled(logrus.Level(level))
 }
 
-func (l *logger) createEntry() *logrus.Entry {
+func (l *logger) createEntry(ctx context.Context) *logrus.Entry {
 	return logrus.
 		NewEntry(l.Logger).
-		WithFields(logrus.Fields{"frame": getFrameInfo()})
+		WithFields(logrus.Fields{
+			"frame":   getFrameInfo(),
+			"context": formatContextData(ctx),
```

Comment:
ik vind formatContextData iets te generiek, het gaat in dit geval alleen maar om het request id. Het zou mooier zijn dat hij gewoon het request id aan de logrus.Fields toevoegt als deze beschikbaar is